### ISSUE TITLE
Crash cause by LWJGL: Unsupported JNI version

### ISSUE THREAD

**Issue by isHarryh** [libgdx/libgdx#7142]
### Issue details
When I *RUN* my LWJGL app in IDEA, it works well, but an warning was outputted by LWJGL:  
`[ThreadLocalUtil] Unsupported JNI version detected, this may result in a crash. Please inform LWJGL developers.`

**Then, if I DEBUG my app in IDEA, the JVM crashed.**

### Reproduction steps
Reproduction steps see Issue details.

I am using JDK 20, so **I think it should be an issue related to the incompatibility between the Java version and the LWJGL version.**

If I degrade my JDK to 18, the crash will not occur any more.

### Version of libGDX
libGDX 1.11.0

### Stacktrace
```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x0000000000000000, pid=15672, tid=5648
#
# JRE version: Java(TM) SE Runtime Environment (20.0.1+9) (build 20.0.1+9-29)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (20.0.1+9-29, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, windows-amd64)
# Problematic frame:
# C  [jdwp.dll+0x1611f]
#
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#
```

#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [ ] HTML/GWT
- [x] Windows
- [ ] Linux
- [ ] macOS

**Comment by Berstanio** [comment]
I think that is a lwjgl3 issue, not a libGDX one.
Also I do think, this has been resolved on lwjgl3 side with lwjgl3 3.3.2. You can either upgrade manually or you could try the latest libGDX snapshot, 1.11.1-SNPASHOT, that depend on the newest lwjgl3 version, if you really need java 20 features/API's.

**Comment by paramidon** [comment]
Just rolled back Java version from 20 to 17 and debug works fine. Thanks
